### DISCUSSION TITLE
Are IDTFs required to post this file?

### DISCUSSION THREAD

**Discussion by BsadlerSRMC** [CMSgov/hospital-price-transparency#16] [Q&A] [answered]
Hello,
We are opening 2 independent testing facilities which will use the hospital TIN but different NPI - will their pricing information be required to be posted for price transparency or are they exempt since they are technically not a hospital?
Thanks!
Brittney

**Comment by shaselton-usds** [comment, marked as answer]
Hospital means an institution, in any State in which State or applicable local law provides for the licensing of hospitals, which is licensed as a hospital pursuant to such law or is approved, by Hospital Price Transparency the agency of such State or locality responsible for licensing hospitals, as meeting the standards established for such licensing. For purposes of this definition, a State includes each of the several States, the District of Columbia, Puerto Rico, the Virgin Islands, Guam, American Samoa, and the Northern Mariana Islands. All hospital location(s) operating under the same hospital license (or approval), such as a hospital’s outpatient department located at an off-campus location (from the main hospital location) operating under the hospital’s license, are subject to the requirements in this rule. Refer to 45 CFR 180.20.
